packages/hop-node/src/watchers/classes/Bridge.ts

```diff
@@ -31,6 +31,8 @@ export default class Bridge extends ContractBase {
   bridgeContract: Contract
   readProvider?: providers.Provider
   bridgeDeployedBlockNumber: number
+  minAmount: BigNumber
```

Comment:
I suggest making these names more descriptive in the context of the bridge; ie `minBondwithdrawalAmount`

Reply:
Resolved with ae73d900762faa1f5898955d07d840716155bc9b